app/models/invalid_letter_address_edipi.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class InvalidLetterAddressEdipi < ActiveRecord::Base
+  validates :edipi, presence: true, uniqueness: true
```

Comment:
Is there any merit to making this validation unique within the scope of each user?  For example, if two users got the same invalid `edipi` we wouldn't know it as written.

Reply:
it's actually the users address that's invalid but CSRA need the user's EDIPI to look up the address to find out why.